tox.ini

```diff
@@ -11,24 +11,32 @@ deps =
   mypy: mypy~=0.711
 
 setenv =
-  PYTHONPATH={toxinidir}/opentelemetry-api/src/
-  mypy: MYPYPATH={env:PYTHONPATH}
+  mypy: MYPYPATH={toxinidir}/opentelemetry-api/src/
 
 changedir =
-  test: opentelemetry-api/tests
+  test-api: opentelemetry-api/tests
+  test-sdk: opentelemetry-sdk/tests
+
+commands_pre =
+  test-api: pip install -e {toxinidir}/opentelemetry-api
+  test-sdk: pip install -e {toxinidir}/opentelemetry-api
+  test-sdk: pip install -e {toxinidir}/opentelemetry-sdk
```

Comment:
It would be better to install these as regular deps so they're not reinstalled on each run. There may be another better way to do this.